docs/book/en/src/cli/wasmedge.md

```diff
@@ -213,7 +214,7 @@ The [hello.js](https://github.com/WasmEdge/WasmEdge/raw/master/examples/js/hello
 You can run:
 
 ```bash
-wasmedge --dir .:. qjs.wasm hello.js 1 2 3
+wasmedge --dir . qjs.wasm hello.js 1 2 3
```

Comment:
We should keep both examples and tell users that they are the same.

Reply:
Done